versions/3.2.0.md

```diff
@@ -3199,6 +3199,7 @@ Field Name | Type | Applies To | Description
 <a name="securitySchemeBearerFormat"></a>bearerFormat | `string` | `http` (`"bearer"`) | A hint to the client to identify how the bearer token is formatted.  Bearer tokens are usually generated by an authorization server, so this information is primarily for documentation purposes.
 <a name="securitySchemeFlows"></a>flows | [OAuth Flows Object](#oauthFlowsObject) | `oauth2` | **REQUIRED**. An object containing configuration information for the flow types supported.
 <a name="securitySchemeOpenIdConnectUrl"></a>openIdConnectUrl | `string` | `openIdConnect` | **REQUIRED**. OpenId Connect URL to discover OAuth2 configuration values. This MUST be in the form of a URL. The OpenID Connect standard requires the use of TLS.
+<a name="securitySchemeoauth2MetadataUrl"></a>oauth2MetadataUrl | `string` | `oauth2` | **OPTIONAL**. A HTTPS URL to discover OAuth2 configuration values. This must be in the form of a URL. See [RFC8414](https://datatracker.ietf.org/doc/html/rfc8414).
```

Comment:
```suggestion
<a name="securitySchemeoauth2MetadataUrl"></a>oauth2MetadataUrl | `string` | `oauth2` |A HTTPS URL to discover OAuth2 configuration values. This must be in the form of a URL. See [RFC8414](https://datatracker.ietf.org/doc/html/rfc8414).
```

The "OPTIONAL" notation seems out of place here. We indicate which are the required fields, but I don't think we usually indicate which are optional.